test/app_signals/traces_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package app_signals
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/xray"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"github.com/aws/amazon-cloudwatch-agent-test/util/awsservice"
+)
+
+const (
+	lookbackDuration     = time.Duration(-80) * time.Minute
+	EKSClusterAnnotation = "HostedIn_EKS_Cluster"
+)
+
+var annotations = map[string]string{
+	"aws_remote_target":      "remote-target",
+	"aws_remote_operation":   "remote-operation",
+	"aws_local_service":      "service-name",
+	"aws_remote_service":     "service-name-remote",
+	"HostedIn_K8s_Namespace": "default",
+	"aws_local_operation":    "operation",
+}
+
+type AppSignalsTracesRunner struct {
+	test_runner.BaseTestRunner
+	testName    string
+	clusterName string
+}
+
+func (t *AppSignalsTracesRunner) Validate() status.TestGroupResult {
+	testResults := []status.TestResult{{
+		Name:   t.testName,
+		Status: status.FAILED,
+	}}
+	timeNow := time.Now()
+	annotations[EKSClusterAnnotation] = t.clusterName
+	xrayFilter := FilterExpression(annotations)
+	traceIds, err := GetTraceIDs(timeNow.Add(lookbackDuration), timeNow, xrayFilter)
```

Comment:
`GetTraceIDs` and `FilterExpression` are already available in the `awsservice` package
https://github.com/aws/amazon-cloudwatch-agent-test/blob/757a5a75c6636cb65ac4f3b1c2c6179ea803f85b/util/awsservice/xray.go#L22

Reply:
Updated in revision